static/app/views/starfish/views/appStartup/screenSummary/countWidget.tsx

```diff
@@ -0,0 +1,131 @@
+import {getInterval} from 'sentry/components/charts/utils';
+import LoadingContainer from 'sentry/components/loading/loadingContainer';
+import {t} from 'sentry/locale';
+import {MultiSeriesEventsStats} from 'sentry/types';
+import {Series, SeriesDataUnit} from 'sentry/types/echarts';
+import {defined} from 'sentry/utils';
+import {tooltipFormatterUsingAggregateOutputType} from 'sentry/utils/discover/charts';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import Chart from 'sentry/views/starfish/components/chart';
+import MiniChartPanel from 'sentry/views/starfish/components/miniChartPanel';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {STARFISH_CHART_INTERVAL_FIDELITY} from 'sentry/views/starfish/utils/constants';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {useEventsStatsQuery} from 'sentry/views/starfish/utils/useEventsStatsQuery';
+import {
+  COLD_START_COLOR,
+  WARM_START_COLOR,
+} from 'sentry/views/starfish/views/appStartup/appStartBreakdown';
+import {OUTPUT_TYPE, YAxis} from 'sentry/views/starfish/views/screens';
+
+const SPAN_OP_TO_STRING = {
+  'app.start.cold': t('Cold Start'),
+  'app.start.warm': t('Warm Start'),
+};
+
+function transformData(data?: MultiSeriesEventsStats) {
+  const transformedSeries: {[yAxisName: string]: Series} = {};
+  if (defined(data)) {
+    Object.keys(data).forEach(yAxis => {
+      transformedSeries[yAxis] = {
+        seriesName: yAxis,
+        data:
+          data[yAxis]?.data.map(datum => {
+            return {
+              name: datum[0] * 1000,
+              value: datum[1][0].count,
+            } as SeriesDataUnit;
+          }) ?? [],
+      };
+    });
+  }
+  return transformedSeries;
+}
+
+interface Props {
+  additionalFilters?: string[];
+}
+
+function CountWidget({additionalFilters}: Props) {
+  const pageFilter = usePageFilters();
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+
+  const query = new MutableSearch([
+    'span.op:[app.start.warm,app.start.cold]',
+    ...(additionalFilters ?? []),
+  ]);
+  const queryString = `${appendReleaseFilters(
+    query,
+    primaryRelease,
+    secondaryRelease
+  )} span.description:["Cold Start","Warm Start"]`;
+
+  const {
+    data: series,
+    isLoading: isSeriesLoading,
+    isError,
+  } = useEventsStatsQuery({
+    eventView: EventView.fromNewQueryWithPageFilters(
+      {
+        name: '',
+        topEvents: '2',
+        fields: ['span.op', 'count()'],
```

Comment:
it might be valuable to show counts broken down by release somewhere on the page - i think it's valuable for the user to know the scales of what they are comparing (ie some signal if they are comparing a realllly low throughput release vs a high throughput one)

Reply:
This is a good point. Olivier brought it up and I think it makes sense too but ugh the chart is going to be too noisy. I'm going to merge this as-is and I'll find a way to represent the counts per release. It might not be so bad as a multi-series widget that spans two rows, but as one it's going to be pretty cramped.